### PR TITLE
Update CHANGES.md and README.md to capture the next release's info

### PR DESCRIPTION
**Pull request by askdre**
During a release, automation will create two commits.

The release commit will
* replace the ## Next header in CHANGES.md with the release version and date
* remove the ## Unreleased Changes section from README.md
* replace ${next-release-tag} in README.md with the release version

The next development commit will
* Add the ## Next section to CHANGES.md
* Replace the release README.md with the development README.md containing the ## Unreleased Changes section and ${next-release-tag} template.
* Update the link to the latest release in the README.md to point to the current release version.

### CODE REVIEW

**Review by davidrabinowitz — CHANGES_REQUESTED** [review]
The README is the main documentation document for the users, and as such it needs to contain the last released version in plain-text, not tag. If memory serves, the original idea was to have a README development doc with the tags in another directory. The release process will copy and transform this doc to the final README in the project root

**Comment by askdre** [comment]
Moved the templated readme to README-template.md which will get instantiated to generate the README.md.  Developers should be notified that any changes made to README.md will be overriden by changes to README-template.md during the release, so they should add their documentation updates to README-template.md.

File: README-template.md [hunk 1]
```diff
@@ -0,0 +1,1140 @@
...
+  .save("dataset.table")
+```
+
+Partition overwriting and the use of `datePartition`, `partitionField` and `partitionType` as described below is not
+supported at this moment by the direct write method.
+
+**Important:** Please refer to the [data ingestion pricing](https://cloud.google.com/bigquery/pricing#data_ingestion_pricing)
+page regarding the BigQuery Storage Write API pricing.
+
+**Important:** Please use version ${next-release-tag} and above for direct writes, as previous
```
Comment (davidrabinowitz) [inline README-template.md:252]: Here the version should be the constant 0.24.2, due to the PR #563 fix

**Review by davidrabinowitz — CHANGES_REQUESTED** [review]

File: README-template.md (hunk 1 above)
Comment (askdre) [reply README-template.md]: Updated line 252 to use the constant instead of the varible.

**Review by davidrabinowitz — APPROVED** [review]

**Pull request merged**